### DISCUSSION TITLE
Icon Request: fa-vegan

### DISCUSSION THREAD

**Discussion by malenagiletti** [FortAwesome/Font-Awesome#21212] [Icon Requests]
Hi,

I hope you can help me. I'm customizing my website, which is Organic restaurants and cafes directory. I'd like to add vegan and vegetarian icons. 

Could you please let me know if you can help me?

Thank you in advance!

Kind Regards
MG
![vegan symbol](https://user-images.githubusercontent.com/37257753/37344152-7b781840-26c1-11e8-8ad7-f75fc65d051b.png)
![vegetarian symbol](https://user-images.githubusercontent.com/37257753/37344158-80289af4-26c1-11e8-9268-9f1ee31bfde7.png)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Vegetarian is a duplicate of #4950, please add a 👍  reaction to that request.

Leaving this open as vegan

Please make sure you [search existing requests](https://github.com/FortAwesome/Font-Awesome/search?type=Issues) before opening new ones

**Comment by CalumChilds** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by samtuckerdavis** [comment]
I'd like to help implement this icon. Here's a design proposal:

## Vegan Icon (fa-vegan)

**Design concept**: The international vegan symbol — a stylized "V" with a small leaf element, inscribed in a circle. This is the most widely recognized vegan indicator globally.

**Design specifications** (consistent with Font Awesome style):
- Clean, geometric "V" letterform
- Small leaf detail on the right arm of the V (2-3 leaf veins for detail at small sizes)
- Optional circle variant (fa-vegan-circle) for badge/label use cases
- Designed to be legible at 14px and sharp at 48px+
- Single-path SVG for optimal rendering

**Use cases** (extensive):
- Restaurant menus and food ordering platforms
- Food delivery apps (UberEats, DoorDash, etc. all need dietary indicators)
- Recipe management apps
- Grocery and shopping list apps
- Nutrition tracking interfaces
- Product labeling systems
- Event catering selection forms

**Market context**: Font Awesome is used on millions of websites. The plant-based food market is projected to reach $77.8B by 2025. Dietary indicator icons are a standard UI requirement that Font Awesome currently can't serve.

Happy to submit an SVG that meets Font Awesome's design guidelines. Is there a preferred design review process for new icon contributions?

---

_Also relevant to the vegetarian icon request (old #4950, now closed): A vegetarian icon could follow the same design language — the Indian FSSAI green dot/brown dot system (mandated on all packaged food in India) provides a well-established visual vocabulary for vegetarian labeling. Happy to design both icons as a pair._
